.github/workflows/linux_job.yml

```diff
@@ -233,6 +222,7 @@ jobs:
           s3-bucket: doc-previews
           if-no-files-found: error
           path: ${{ env.RUNNER_DOCS_DIR }}
+          # ${{ env.repository }} is $OWNER/$REPO
```

Comment:
Ok, I was wrong above. Since this contains the owner and repo, the legacy label was correctly attached.